docs/integrations/enterprise-connectors/source-oracle.md

```diff
@@ -155,69 +205,81 @@ by the `authorized_keys` file on your bastion host. The public key should be add
 host to whichever user you want to use with Airbyte. The private key is provided via copy-and-paste
 to the Airbyte connector configuration screen, so it may log in to the bastion.
 
-## Change Data Capture (CDC)
+## Change Data Capture (CDC) limitations
+
+The Enterprise Oracle source connector supports incremental syncs using Change Data Capture (CDC) with some limitations.
```

Comment:
Once you've initialized CDC as you have already in the heading, you can just use "CDC" going forward.